sketch_map_tool/database/client.py

```diff
@@ -54,8 +57,7 @@ def _select_id_map(uuid) -> dict:
         raw = curs.fetchall()
     if raw:
         return raw[0][0]
-    else:
-        raise UUIDNotFoundError("There are no tasks in the broker for UUID: " + uuid)
+    raise UUIDNotFoundError("There are no tasks in the broker for UUID: " + uuid)
```

Comment:
why did you remove the else statement?

Reply:
For style reasons, restored previous version